### PR TITLE
Updates config to use right version of drush, adds initial 7.4 config.

### PR DESCRIPTION
**Pull request by synterra**
I had to edit `getconfig.sh` to change the drush version, as `drush` defaults to drush 9 instead of 8.

I also added a `-r` flag to get the SQL command working.

### CODE REVIEW

File: getconfig/getconfig.php [hunk 1]
```diff
@@ -0,0 +1,178 @@
+<?php
```
Comment (grugnog) [inline getconfig/getconfig.php]: I think this was committed by mistake - it lives in it's own repo, so if we need to make changes we should make them there.

File: httpd/Dockerfile-e [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM httpd:2.4.18
```
Comment (grugnog) [inline httpd/Dockerfile-e]: Are the `-e` files backups?

**Review by grugnog — CHANGES_REQUESTED** [review]
Looks good from a config perspective - I think some extra files were added though. 
We would need to add tests in Jenkinsfile to see if it builds

**Comment by synterra** [comment]
New PR for this is at https://github.com/drydockcloud/drupal-acquia/pull/16

**Pull request closed without merging**